<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="pkg_name" value="vehicle_detection"/>
    <arg name="node_name" default="vehicle_filter_node"/>

    <group ns="$(arg veh)">        
        <node  name="vehicle_filter_node" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen"/>
    </group>

    <!-- Publications -->

    <!-- Subscriptions -->

</launch>

